docker-compose.yml

```diff
@@ -12,6 +12,7 @@ x-env: &x-env
   NOTIFICATION_REDIS_HOST: redis
   REDIS_HOST: redis
   SESSION_DRIVER: redis
+  SCHEMA: "${SCHEMA:-1}"
```

Comment:
this should be in the service itself as the name is too generic for a very specific thing.